examples/jsm/exporters/USDZExporter.js

```diff
@@ -53,7 +53,7 @@ class USDZExporter {
 		} );
 
 		output += buildMaterials( materials );
-		output += buildTextures( textures );
+		//output += buildTextures( textures );
```

Comment:
Do not build all textures in same scope, their defs are now in side each material.